CHANGELOG.md

```diff
@@ -4,13 +4,9 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
-- Added _Experimental_ `RouteRefreshStatesObserver` that can be used to observe route refresh states. To subscribe and unsubscribe on updates corresponding use `MapboxNavigation#registerRouteRefreshStateObserver` and `MapboxNavigation#unregisterRouteRefreshStateObserver`. [#6345](https://github.com/mapbox/mapbox-navigation-android/pull/6345)
```

Comment:
@SevaZhukov looks like the tool didn't clean up the changelog with last release.

Reply:
cc @dzinad for visibility as this also tripped up the changelog script

Reply:
As I was saying in Kyle's PR, the check is more for day-to-day cases when you add something in the wrong place. If you update changelog for release, it will fail (it would be very difficult to set up the check in a way that it would distinguish all the cases). In rare cases like this one (we don't usually move the entries) you just ignore the check. It's better if it fails when it shouldn't then if it skips some error. That was my vision when I was adding all the rules.
If you feel differently, let's discuss but I think it's fine as is.

Reply:
Thanks @dzinad, that makes sense :+1: